synapse/storage/schema/__init__.py

```diff
@@ -66,10 +66,11 @@
       updated.
 
 Changes in SCHEMA_VERSION = 70:
-    - event_reference_hashes is no longer written to.
+    - event_reference_hashes is no longer written to. 
 
 Changes in SCHEMA_VERSION = 71:
     - event_edges.room_id is no longer read from.
+    - Tables related to groups are no longer accessed.
```

Comment:
I think this is the correct note to make? In the future we'll set the `SCHEMA_COMPAT_VERSION` to 71 to enforce this?